### DISCUSSION TITLE
What high school did you graduate from?

### DISCUSSION THREAD

**Discussion by Kotoomii** [NAIST-SE/AttractingContributors#1075] [2024 Attraction Contributions]
Please tell me which high school you graduated from.

https://github.com/Kotoomii/AttractingContributors_YourHighSchool

**Comment by gill876** [comment]
I went to William Knibb Memorial High School in Jamaica.
The same high school Usain Bolt used to attend!

Please contribute to my project!
https://github.com/NAIST-SE/AttractingContributors/discussions/1089#discussion-6846489
